users/ritika-desai-2.txt

```diff
@@ -0,0 +1,2 @@
+ - OIT
```

Comment:
What does this acronym stand for? Please add the fully written version of the acronym to this line